timerMgr.js

```diff
@@ -73,7 +73,8 @@ class TimerMgr {
       this.timers[key] = {
         'timer': timer,
         'flapping': flappingFunc,
-        'notFlapping': (func) ? func : this.#noop
+        'notFlapping': (func) ? func : this.#noop,
+		'when': timeout,
```

Comment:
Please use two sopaces per indent instead of tabs to remain consistent with the surrounding code.